gtk4-macros/tests/templates.rs

```diff
@@ -199,3 +199,57 @@ glib::wrapper! {
 fn template_child_without_attribute() {
     let _: MyWidget3 = glib::Object::new(&[]);
 }
+
+mod imp4 {
+    use super::*;
+
+    #[derive(Debug, Default, CompositeTemplate)]
+    #[template(string = "
+    template MyWidget4 : Widget {
```

Comment:
blueprint is not installed on the CI so this would currently fail :) Can you add it?